core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt

```diff
@@ -6,6 +6,12 @@ package software.aws.toolkits.core.credentials
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider
 
 interface CredentialIdentifier {
```

Comment:
Should probably make a comment that this should *not* be extended directly but that implementers should extend the base.

Reply:
Added